### PR TITLE
Remove walls of text from issue templates

### PR DESCRIPTION
**Pull request by FoRVaiS**
## Summary
Remove explanations from templates and save them into an [archive](https://github.com/Geek-Collectors-Network/geek-collectors-network/wiki/GitHub-Issue-Templates) in the wiki.

### CODE REVIEW

**Comment by tgrigoruk** [comment]
May I suggest something in between - just a short description of what each section should be about. (Also I prefer h3 over h2, the line is visual clutter to me).
For example (first two sections of PR template):

### Summary

_Concise overview of changes being made_

### Retrospective

_Challenges overcome, insights made, strategies learned, etc._

File: .github/PULL_REQUEST_TEMPLATE.md
Comment (tgrigoruk) [inline .github/PULL_REQUEST_TEMPLATE.md:1]: May I suggest something in between - just a short description of what each section should be about. (Also I prefer h3 over h2, the line is visual clutter to me).
For example (first two sections of PR template):

### Summary

_Concise overview of changes being made_

### Retrospective

_Challenges overcome, insights made, strategies learned, etc._

**Review by tgrigoruk — CHANGES_REQUESTED** [review]
May I suggest something in between - just a short description of what each section should be about. (Also I prefer h3 over h2, the horizontal line that h2 creates is visual clutter to me).
For example (first two sections of PR template):

### Summary

_Concise overview of changes being made_

### Retrospective

_Challenges overcome, insights made, strategies learned, etc._

**Comment by tgrigoruk** [comment]
I'm not sure if any changes were made since my last comment. If you don't agree with keeping a single concise line of text for each section, fine, but I'm pretty adamant about using ### over ##. There's enough horizontal lines on any given github page as it is.

**Comment by FoRVaiS** [comment]
@tgrigoruk Changes made.

### Summary
 - Use h3 instead of h2
 - Add one-line descriptions for each section
 - Removed "solution" section from "new task" template

**Review by tgrigoruk — APPROVED** [review]

**Pull request merged**